sql/core/src/main/scala/org/apache/spark/sql/execution/commands.scala

```diff
@@ -418,3 +419,312 @@ case class SetDatabaseCommand(databaseName: String) extends RunnableCommand {
 
   override val output: Seq[Attribute] = Seq.empty
 }
+
+case class CreateDataBase(
+    databaseName: String,
+    allowExisting: Boolean,
+    path: Option[String],
+    comment: Option[String],
+    props: Map[String, String])(sql: String) extends RunnableCommand with Logging {
+
+  override def run(sqlContext: SQLContext): Seq[Row] = {
```

Comment:
`override def run(sqlContext: SQLContext)` and `override val output: Seq[Attribute]` are the same is every command. Lets introduce a class which implements this for us.